src/main/java/org/jsoup/safety/Cleaner.java

```diff
@@ -90,6 +90,16 @@ public boolean isValidBodyHtml(String bodyHtml) {
         return numDiscarded == 0 && errorList.isEmpty();
     }
 
+    public boolean isValidBodyHtml(String bodyHtml,String baseurl) {
```

Comment:
DRY - make the first isValidBodyHtml call the second:

`public boolean isValidBodyHtml(String bodyHtml) {
        return isValidBodyHtml(bodyHtml, "");
    }`